plugins/source/azurev2/Makefile

```diff
@@ -0,0 +1,31 @@
+# Install tools
+.PHONY: install-tools
+install-tools:
+	@echo Installing tools from tools/tool.go
+	@cat tools/tool.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
+
+# Install pre-commit hooks. This requires pre-commit to be installed (https://pre-commit.com/)
+.PHONY: install-hooks
+install-hooks:
+	pre-commit install
+
+.PHONY: gen-code
+gen-code:
+	go install golang.org/x/tools/cmd/goimports@latest
+	go run codegen/main.go
+
+# Generate mocks for mock/unit testing
+.PHONY: gen-mocks
+gen-mocks:
+	go generate ./client/services/...
+
+# Generate docs
+.PHONY: gen-docs
+gen-docs:
+	rm -rf ./docs/tables/*
+	go run main.go doc ./docs/tables
+
+
+# All gen targets
+.PHONY: gen
+gen: gen-code gen-mocks
```

Comment:
This also needs `gen-docs`

Reply:
yep, I'll add it once I start gen the resources